docs/source/rbac/roles.md

```diff
@@ -37,10 +37,10 @@ When a new user gets created, they are assigned their default role, `user`. Addi
 Services do not have a default role. Services without roles have no access to the guarded API end-points. So, most services will require assignment of a role in order to function.
 
 **Groups** \
-A group does not require any role, and has no roles by default. If a user is a member of a group, they automatically inherit any of the group's permissions (see {ref}`resolving-roles-scopes-target` for more details). This is useful for assigning a set of common permissions to several users.
+A group does not require any role, and have no role by default. If a user is a member of a group, they automatically inherit any of the group's permissions (see {ref}`resolving-roles-scopes-target` for more details). This is useful for assigning a set of common permissions to several users.
```

Comment:
This change is incorrect. It is "A group has no roles" not "A group have no role"